Docs/user_guide/modules/retiredmodules.md

```diff
@@ -5,6 +5,9 @@ The modules listed below have been retired and are no longer present in 3D Slice
 Retiring modules reduces burden on Slicer developers. Most common reason for retiring a module is that a new, improved module is introduced.
 A module may also be retired if it no longer works properly (for example due to changes in Slicer core or third-party libraries) and it is not worth investing time into fixing the issue, because the module is not widely used or alternatives exist.
 
+- **Annotations**
+  - Retired: 2023-01-02, [commit 3c3fb74](https://github.com/Slicer/Slicer/commit/3c3fb74ac329ae2916fa3549699fbc49096ca179)
```

Comment:
I’m pretty sure this commit hash is going to change once you rebase and merge these commits into `main`. I think you’ll need a follow up commit once this PR is merged to get the correct hash on the `main` branch.

Reply:
Good point, thanks. I'll update the commit hash after the merge.

Reply:
Updated in https://github.com/Slicer/Slicer/commit/017b64f1f562a2e40b47b903df6c0daba6ad08e0